docs/source/conf.py

```diff
@@ -70,8 +75,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Happi Deck'
-copyright = '2017, SLAC National Accelerator Laboratory'
+project = 'Happi'
+copyright = '2020, SLAC National Accelerator Laboratory'
```

Comment:
```suggestion
copyright = '2021, SLAC National Accelerator Laboratory'
```

Reply:
Now that's embarrassing 😁

Reply:
Hey man, is 2020 really over? Nobody can say.